pkg/providers/kafka/model_source.go

```diff
@@ -10,21 +10,21 @@ import (
 const DefaultAuth = "admin"
 
 type KafkaSource struct {
-	Connection  *KafkaConnectionOptions
-	Auth        *KafkaAuth
-	Topic       string
-	GroupTopics []string
-	Transformer *model.DataTransformOptions
+	Connection  *KafkaConnectionOptions     `json:"connection"`
```

Comment:
this one is more dangerouse, having exact json-tag will make this model more strict, right now prop-names are registry insensitive, having such change would make migration to a newer version of transfer a nightmare, better to rollback.